test/fixtures/is-plain-object/case-1/result.js

```diff
@@ -0,0 +1,21 @@
+import assert from 'assert';
+
+// Test cases
+assert.strictEqual((typeof Object.create({}) === "object" && (Object.create({}).constructor === Object || Object.create({}).constructor === undefined)), true);
+assert.strictEqual((typeof Object.create(Object.prototype) === "object" && (Object.create(Object.prototype).constructor === Object || Object.create(Object.prototype).constructor === undefined)), true);
+assert.strictEqual((typeof { foo: 'bar' } === "object" && ({ foo: 'bar' }.constructor === Object || { foo: 'bar' }.constructor === undefined)), true);
```

Comment:
Cashing should be applied, for example, for object literals too. They could have some logic like computed properties.

Reply:
Ahh, I get what you mean. We should add some sort of caching into the codemod itself, right?